remoteit/desktop
# [DESK-737] Rename devices with legacy API

### Description


Rename devices with legacy API 
Service edit form refactored from inline edit to standard form.

### Test plan



1. Sign in
2. Go to a device you own or have shared to you 
3. Click edit service
4. You should be able to change the service name
5. Go to "this device"
6. Should be able to rename the device
7. Should be able to rename the services and change the service details

### Ticket



<https://remoteit.atlassian.net/browse/DESK-737>
